server/game/cards/01_SOR/units/SuperlaserTechnician.ts

```diff
@@ -0,0 +1,24 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { Location } from '../../../core/Constants';
+
+export default class SuperlaserTechnician extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '8954587682',
+            internalName: 'superlaser-technician'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenDefeatedAbility({
+            title: 'Put Superlaser Technician into play as a resource and ready it',
+            optional: true,
+            immediateEffect: AbilityHelper.immediateEffects.sequential([
```

Comment:
Should be `simultaneous`

Reply:
Are you sure? I would think that it would have to enter as a resource before it could be readied. Simply swapping in simultaneous causes the test to fail.